images/linux/scripts/installers/Install-Toolset.ps1

```diff
@@ -55,4 +57,6 @@ foreach ($tool in $tools) {
 
 chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/Python
 chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/node
-chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/go
\ No newline at end of file
+chown -R "$($env:SUDO_USER):$($env:SUDO_USER)" /opt/hostedtoolcache/go
+
+Invoke-PesterTests -TestFile "Toolset" -TestName "Toolset"
```

Comment:
I would suggest to run tests in `Configure-Toolset.ps1`

Reply:
And no need to specify TestName here, it is the only test suite in the file

Reply:
Thank you for your suggestions. I would prefer to leave it as it is. We install tools in this script, so it make sense to invoke tests from here as well.

Reply:
But earlier we had actions: `install` -> `configure` -> `validate`. Now, this chain will be changed to `install` -> `validate (run tests)`-> `configure`.

Reply:
Good point, @vsafonkin, thank you! Moved test invocation to `Configure-Toolset.ps1`.